src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -352,14 +352,14 @@ private String buildLookupEndpoint(String type, Object... params) {
       .getDetails()
       .getProductIds()
       .stream()
-      .map(productId -> productId.getProductIdType().value())
```

Comment:
I believe the `getProductTypesMap` method is not needed anymore. The main purpose of it was to get identifier ids based on names. The changes in scope of this ticket to replace name by id so no extra calls needed to inventory-storage.
After the current updates the method returns map where key and value are absolutely the same

Reply:
Please also remove `identifier-types` interface dependency along with related permissions

Reply:
I have removed the method and updated tests.